spec/models/dimensions/item_spec.rb

```diff
@@ -2,4 +2,41 @@
 
 RSpec.describe Dimensions::Item, type: :model do
   it { is_expected.to validate_presence_of(:content_id) }
+
+  let(:test_json) do
+    {
+      schema_name: "answer",
+      details: {
+        body: "This is a test"
+      }
+    }.to_json
+  end
+
+  let(:expected_result) do
+    "This is a test"
+  end
+
+  describe "#get_content" do
+    it "returns nil if json is empty" do
+      item = create(:dimensions_item, raw_json: {}.to_json)
+      expect(item.get_content).to eq(nil)
+    end
+
+    it "returns content json if it is a valid schema and details.body exist" do
```

Comment:
I would create a `context` to consider both scenarios: when valid schema, and non valid schema

Something like:

```ruby
context 'when valid schema' do
end

context 'when invalid schema' do
end
```

This will remove from your test description the `and` clause.